dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md

```diff
@@ -110,15 +110,74 @@ First, make sure that Docker is running. Then, you can create a local Kubernetes
 
 ## Step 2: Deploy TiDB Operator in the Kubernetes cluster
 
-Refer to the steps in [Deploy TiDB Operator](/dev/tidb-in-kubernetes/deploy/tidb-operator.md#install-tidb-operator).
+1. Install Helm and configure it with the charts provided by PingCAP. Refer to the steps in [Use Helm](/dev/tidb-in-kubernetes/reference/tools/in-kubernetes.md#use-helm).
+
+2. Deploy TiDB Operator. Refer to the steps in [Deploy TiDB Operator](/dev/tidb-in-kubernetes/deploy/tidb-operator.md#install-tidb-operator)
 
 ## Step 3: Deploy a TiDB cluster in the Kubernetes cluster
 
 Refer to the steps in [Deploy TiDB on General Kubernetes](/dev/tidb-in-kubernetes/deploy/general-kubernetes.md#deploy-tidb-cluster).
 
 ## Access the database and monitoring dashboards
 
-Refer to the steps in [View the monitoring dashboard](/dev/tidb-in-kubernetes/monitor/tidb-in-kubernetes.md#view-the-monitoring-dashboard).
+To access TiDB clusters, use `kubectl port-forward` command to expose services to the host. The port number used in command are in `<host machine port>:<k8s service port>` format.
+
+- Access TiDB using the MySQL client
+
+    Before you start testing your TiDB cluster, make sure you have installed a MySQL client.
+
+    1. Use kubectl to forward the host machine port to the TiDB service port:
+
+        {{< copyable "shell-regular" >}}
+
+        ``` shell
+        kubectl port-forward svc/<release-name>-tidb 4000:4000 --namespace=<namespace>
+        ```
+
+        If results like `Forwarding from 0.0.0.0:4000 -> 4000` is returned, it means the host machine port is successfully forwarded.
+
+    2. Access TiDB using the MySQL client, and open a **new** terminal tab or window to run the following command:
+
+        {{< copyable "shell-regular" >}}
+
+        ``` shell
+        mysql -h 127.0.0.1 -P 4000 -u root
+        ```
+
+        When testing finishes, use <kbd>Ctrl</kbd>+<kbd>C</kbd> to exit.
+
+- Access TiDB through the monitoring dashboard
+
+    1. Use kubectl to forward the host machine port to the Grafana service port:
+
+        {{< copyable "shell-regular" >}}
+
+        ``` shell
+        kubectl port-forward svc/<release-name>-grafana 3000:3000 --namespace=<namespace>
+        ```
+
+        If results like `Forwarding from 0.0.0.0:4000 -> 4000` is returned, it means the host machine port is successfully forwarded.
+
+    2. Open your web browser at <http://localhost:3000> to access the Grafana monitoring dashboard.
+
+        - default username: admin
+        - default password: admin
+
+        When testing finishes, use <kbd>Ctrl</kbd>+<kbd>C</kbd> to exit.
+
+    > **Note:**
+    >
+    > If you are deploying kind on a remote machine rather than a local PC, there might be problems connecting to "localhost".
```

Comment:
```suggestion
    > If you are deploying kind on a remote machine rather than a local PC, there might be problems accessing the monitoring dashboard of the remote system through "localhost".
```